plugins/doc_fragments/keycloak.py

```diff
@@ -41,15 +41,11 @@ class ModuleDocFragment(object):
         description:
             - Username to authenticate for API access with.
         type: str
-        aliases:
-          - username
```

Comment:
Please note that this is a breaking change that can not be merged.

Reply:
Shall we consult with other developers, because this change is mandatory in order to make user module work, in all the other resources for Keycloak, the resource representation uses name parameter, but only for the user module it is username, which conflicts with the same alias being set for auth_username, and moreover we need clear differentiation of username for user creation and auth_username for API authentication towards keycloak endpoint.

Reply:
There is nothing to consult. Removing this option for the existing modules is a breaking change and that cannot be merged.

Reply:
(Nobody is stopping you from calling the module parameters differently than the fields are called in Keycloak. It is unfortunate that `username` and `password` is already used differently for the other modules, but you cannot simply change that for the other modules just because these names are more convenient for something else in a new module.)

Reply:
Noted

Reply:
@felixfontein reverted the breaking changes in latest commits.